src/main/java/io/jenkins/plugins/sprp/YAML_FlowDefinition.java

```diff
@@ -125,6 +138,36 @@ public FlowExecution create(FlowExecutionOwner owner, TaskListener listener,
         return new CpsFlowExecution(script, false, owner);
     }
 
+    private List<String> getCleanRefspecs(List<UserRemoteConfig> userRemoteConfigs){
```

Comment:
What this method does is not clear to me, in other words, what is the meaning of a "clean" refspec (is there such a thing as an unclean refspec?). It would be good to have a javadoc that describes what this method does. It should show an example of an input along with the output being produced.

Reply:
In refspecs there will be '+' symbol and some trailing white spaces, we need to remove them.

Reply:
Yes, I understand your answer, but you need to say what the method does in the javadoc because the code is not obvious in this case. The Twitter Java style guide has an [example of how to use javadoc for a method](https://github.com/twitter/commons/blob/master/src/java/com/twitter/common/styleguide.md#documenting-a-method).